osu.Game/Skinning/LocalSkinOverrideContainer.cs

```diff
@@ -82,13 +82,13 @@ protected override IReadOnlyDependencyContainer CreateLocalDependencies(IReadOnl
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuConfigManager config)
+        private void load(BindableVisualSettings visualSettings)
         {
-            beatmapSkins = config.GetBindable<bool>(OsuSetting.BeatmapSkins);
+            beatmapSkins = visualSettings.BeatmapSkins;
```

Comment:
In general, we don't ever want to keep non-weak-reference upwards references like how it's done here and in the other classes that use visualSettings this way.